style.css

```diff
@@ -512,10 +618,48 @@ textarea {
     margin: 0 auto;
   }
 
+  .pop-up--container {
+    max-width: 580px;
+    margin: 0 auto;
+  }
+
+  .pop-up--container > .mobile {
+    display: none;
+  }
+
+  .pop-up--container > .desktop {
+    display: block;
+  }
+
+  .pop-up--container > .range > .mobile {
+    display: none;
+  }
+
+  .pop-up--container > .range > .desktop {
+    display: block;
+  }
+
+  .pop-up--container > .desktop > img {
+    width: 600px;
+  }
+
+  .pop-up__header > .project-name {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .pop-up__footer > .options {
+    width: 50%;
+  }
+
   .work-section__box > .work-section__card {
     flex-direction: row;
   }
 
+  .pop-up--container.active {
+    margin-bottom: 30px;
+  }
+
```

Comment:
## [OPTIONAL]

- [ ] NICE JOB 👏  sure your popup design is looking nice 💯  but only one last thing is to add also a margin-top to your popup in your desktop version so that it may look like the original design. Check here 👇  for more clarifications 

The original popup | your popup 
---------|-------
![image](https://user-images.githubusercontent.com/65068771/151337621-00a40dba-5234-456b-9a9d-66dc329cfafb.png) | ![image](https://user-images.githubusercontent.com/65068771/151337972-50d13569-5d97-4ff1-ad9a-3051f490a88b.png)